dev/Lights/MaterialHelper.cpp

```diff
@@ -27,11 +27,7 @@ void MaterialHelperBase::SimulateDisabledByPolicy(bool value)
     if (oldValue != value)
     {
         instance->m_simulateDisabledByPolicy = value;
-#if BUILD_WINDOWS
-        instance->NotifyAdditionalPolicyChangedListeners();
-#else
         instance->UpdatePolicyStatus();
```

Comment:
@DmitriyKomin is going to need all this code eventually to make this work in WinUI 3.0. I think leaving it might be more helpful so that when we get it hooked up finally it doesn't look like a bunch of new code?

Reply:
I'll revert all the changes to Lights and Materials